services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchSpec.groovy

```diff
@@ -24,4 +33,118 @@ class SwitchSpec extends BaseSpecification {
         def exc = thrown(HttpClientErrorException)
         exc.rawStatusCode == 400
     }
+
+    def "Delete nonexistent switch"() {
```

Comment:
If possible, name the test with what expected behavior is being tested, instead of what actions are being done.
"Delete active switch" -> "Cannot delete active switch"
"Force switch delete" -> "Can delete active switch if using force delete"